app/actions/balancesActions.js

```diff
@@ -69,9 +69,12 @@ function determineIfBalanceUpdated(
 
 async function getBalances({ net, address }: Props) {
   const { soundEnabled, tokens } = await getSettings()
-  const network = findNetworkIdByLabel(net)
+  const network = findNetworkByLabel(net)
 
   let endpoint = await getNode(net)
+  if (isEmpty(endpoint)) {
```

Comment:
@evgenyboxer if you refer to the documentation for electron-json-storage, when the key does not exist and empty object is returned https://www.npmjs.com/package/electron-json-storage#storagegetkey-options-callback thus `isEmpty` works really well here because:
- On a clean install the node will always be an empty object
- If the user manually selects "automatic" node selection we save an empty string for the key